data/mods/gen2/scripts.js

```diff
@@ -245,8 +245,6 @@ let BattleScripts = {
 			move.totalDamage = damage;
 		}
 		if (move.category !== 'Status') {
-			// FIXME: The stored damage should be calculated ignoring Substitute.
```

Comment:
I'm pretty sure this doesn't apply at all to Gen 2 and the comment was simply copied.